lib/zipsign/zip.cc

```diff
@@ -5,12 +5,10 @@
 #include "zipsign/zip.hpp"
 #include "zipsign/file.hpp"
 
+#include <algorithm>
```

Comment:
Why was algorithm moved to the top?
I usually prefer the following order: local headers, c headers, c++ headers.
I'm not strict about this myself, but I'm interested whether there is any reason to move algorithm header.

Reply:
Alright than, I may change the order later ;-)